em_workflows/dm_conversion/flow.py

```diff
@@ -92,7 +90,7 @@ def convert_if_int16_tiff(file_path: FilePath) -> None:
         str(tif_8_bit),
     ]
     utils.log(f"Generated cmd {cmd}")
-    FilePath.run(cmd, log_fp)
+    FilePath.run(cmd, log_fp, env={"IMOD_OUTPUT_FORMAT": "TIF"})
```

Comment:
oh, cool

Reply:
The important things added is the usage of pipes with streaming output so that the output of the process can be seen live or if the task is killed by slum the partial output would be there.